examples/benchmarks/bubblebench/simple-tree/src/schema.ts

```diff
@@ -0,0 +1,64 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import type { IBubble, ITreeClient } from "@fluid-example/bubblebench-common";
+import { SchemaFactory, TreeConfiguration } from "@fluidframework/tree";
+
+const sf = new SchemaFactory("bubble-bench-simple-tree");
+
+export class Bubble
+	extends sf.object("Bubble", {
+		x: sf.number,
+		y: sf.number,
+		r: sf.number,
+		vx: sf.number,
+		vy: sf.number,
+	})
+	implements IBubble
+{
+	constructor(public readonly bubble: IBubble) {
+		super(bubble);
+	}
```

Comment:
```suggestion
```

Reply:
Storing a copy of the object used to initialize this in a public field seems like a bad idea, since that data is redundant, but also can get out of date.